app/views/spotlight/exhibits/_exhibit_card.html.erb

```diff
@@ -1,7 +1,11 @@
 <div class="col-xs-12 col-sm-4 exhibit-card">
   <div class="flipper">
     <div class="card-front card-face">
-      <%= image_tag(exhibit.thumbnail.image.square.url) if exhibit.thumbnail.present? %>
+      <% if exhibit.thumbnail.present? %>
+        <%= image_tag(exhibit.thumbnail.image.square.url) %>
+      <% else %>
+        <%= image_tag 'spotlight/default_thumbnail.jpg' %>
```

Comment:
Overriding the image downstream should be fine, but I haven't tried it. I'll try to pull the front and back of the card out into separate partials and write a simple view spec for it.